Documentation/batch-scripts/mentoring.py

```
@@ -86,6 +86,17 @@ def update_last_run(entity: str, ts: datetime):
     with open(STATE_FILE, "w") as f:
         json.dump(state, f, indent=2)
 
+# ---------------- Epoch to UTC ----------------
+def convert_epoch_to_utc(epoch_value):
+    if not epoch_value:
+        return None
+    try:
+        dt = datetime.fromtimestamp(int(epoch_value), tz=timezone.utc)
+        return dt.strftime("%Y-%m-%d %H:%M:%S")
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)